### PR TITLE
removing resource group name from PrepareNetworkPoliciesRequest

### PR DESCRIPTION
**Pull request by mvvsubbu**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [x] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by number213** [comment]
Please ignore `NetworkValidation`, the failure is expected as some changes have not been merged in PowerShell's master yet

**Comment by mvvsubbu** [comment]
@tjprescott do I need to investigate  ~ Ignore Test-only (rest-api-specs) (Spellcheck) failures and SDK generation failures

File: specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/virtualNetwork.json [hunk 1]
```diff
@@ -1674,10 +1674,6 @@
           "type": "string",
           "description": "The name of the service for which subnet is being prepared for."
         },
-        "resourceGroupName": {
```
Comment (KrisBash) [inline specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/virtualNetwork.json]: Has this version shipped? Isn't this breaking?

**Review by KrisBash — CHANGES_REQUESTED** [review]
Please see comment

File: specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/virtualNetwork.json (hunk 1 above)
Comment (mvvsubbu) [reply specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/virtualNetwork.json]: @ambhatt can you answer

Comment (mvvsubbu) [reply specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/virtualNetwork.json]: @Am018 Can you answer this?

Comment (Am018) [reply specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/virtualNetwork.json]: There was no work done (Powershell, CLI) on top of this implementation and this API is still behind feature flag and hence not getting used by anyone other than some internal MSFT teams for testing. Hence its a safe change.

**Review by Am018 — APPROVED** [review]

File: specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/virtualNetwork.json (hunk 1 above)
Comment (MikhailTryakhov) [reply specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/virtualNetwork.json]: for feature flags we do strongly recommend to use private previews, not public one

Comment (mvvsubbu) [reply specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/virtualNetwork.json]: We are adding these changes in new api version (2019-06-01) folder isn't that good enough?

Comment (mvvsubbu) [reply specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/virtualNetwork.json]: @KrisBash  we are making these changes in new API version (2019-06-01). So it shouldn't be any issue. Please let me know if there is anything else I need to do for this PR.

Comment (mvvsubbu) [reply specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/virtualNetwork.json]: @MikhailTryakhov I had discussion with @Am018 and this API is available publicly but not used that's it. So we decided to make changes in new API version 2019-06-01 and publish changes in public repository only. So please take a look at the PR and approve it.

**Review by tjprescott — APPROVED** [review]

**Review by KrisBash — APPROVED** [review]

**Pull request merged**